runtime/src/lib.rs

```diff
@@ -612,26 +644,125 @@ parameter_types! {
         .saturating_sub(BlockExecutionWeight::get());
 }
 
+pub struct BaseFeeThreshold;
```

Comment:
Some comments explaining these would be helpful, like what are lower, ideal, elasticity, etc and why these values?/

Reply:
Added docs

Reply:
Thanks